openslides/motions/static/templates/motions/motion-import.html

```diff
@@ -0,0 +1,140 @@
+<div class="header">
+  <div class="title">
+    <div class="submenu">
+      <a ui-sref="motions.motion.list" class="btn btn-sm btn-default">
+        <i class="fa fa-angle-double-left fa-lg"></i>
+        <translate>Back to overview</translate>
+      </a>
+    </div>
+    <h1 translate>Import motions</h1>
+  </div>
+</div>
+
+<div class="details">
+
+  <div class="block row">
+    <div class="title">
+      <h3 translate>Select a CSV file
+    </div>
+    <div class="block right import">
+      <label class="label" for="inputSeparator" translate>Separator</label>
+      <input type="text" ng-model="separator" ng-change="setSeparator()" ng-init="separator=separator" id="inputSeparator">
+      <br>
+      <label class="label" for="selectEncoding" translate>Encoding</label>
+      <select ng-model="encoding" ng-options="enc as enc for enc in encodingOptions"
+          ng-selected="setEncoding()" ng-init="encoding=encoding" id="selectEncoding"></select>
+      <ng-csv-import
+          content="csv.content"
+          header="csv.header"
+          header-visible="csv.headerVisible"
+          separator="csv.separator"
+          separator-visible="csv.separatorVisible"
+          result="csv.result"
+          encoding="csv.encoding"
+          encoding-visible="csv.encodingVisible"></ng-csv-import>
+    </div>
+  </div>
+
+  <h4 translate>Please note:</h4>
+  <ul><!--TODO: utf-8 encoding still required with angular-csv? -->
+      <li><translate>Required comma or semicolon separated values with these column header names in the first row</translate>:<br>
+          <code translate>'identifier, title, text, reason, submitter, category'</code>
+      <li translate>Identifier, reason, submitter and category are optional and may be empty.
+      <li translate>Only double quotes are accepted as text delimiter (no single quotes).
+      <li><a href="https://github.com/OpenSlides/OpenSlides/wiki/CSV-Import" translate>
+            Use the CSV example file from OpenSlides Wiki.
+          </a>
+  </ul>
+
+  <div ng-if="csv.result">
+    <h3 translate>Preview</h3>
+    <table class="table table-striped table-bordered table-condensed">
+      <thead>
+        <tr>
+          <th>
+          <th>#
+          <th translate>Identifier
+          <th translate>Title
+          <th translate>Text
+          <th translate>Reason
+          <th translate>Submitter
+          <th translate>Category</th>
+      <tbody ng-repeat="motion in motions">
+        <tr>
+          <td class="minimum"
+              ng-class="{ 'text-danger': motion.importerror, 'text-success': motion.imported }">
```

Comment:
You can do it if you like. But we/you have to merge this pull request tomorrow so that we can release 2.0b1 on monday!